plugins/sigma.layout.dagre/sigma.layout.dagre.js

```diff
@@ -89,6 +84,12 @@
    * ------------------
    */
   function dagreLayout() {
+    if (typeof dagre === 'undefined')
```

Comment:
I'd prefer to throw error early if the file is included by the dependency is missing.